hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestObserverWithRouter.java

```diff
@@ -439,4 +446,38 @@ public void testRouterMsync() throws Exception {
     assertEquals("Four calls should be sent to active", 4,
         rpcCountForActive);
   }
+
+  @Test
```

Comment:
Could you add
> @Tag(SKIP_BEFORE_EACH_CLUSTER_STARTUP)

so this test doesn't do the MiniDFSCluster setup. Runtime goes down from 21s to less than a second on my laptop.